docs/source/providers/inference/index.md

```diff
@@ -7,6 +7,7 @@ Llama Stack Inference API for generating completions, chat completions, and embe
     This API provides the raw interface to the underlying models. Two kinds of models are supported:
     - LLM models: these models generate "raw" and "chat" (conversational) completions.
     - Embedding models: these models generate embeddings to be used for semantic search.
+    - Rerank models: these models rerank the documents by relevance. 
```

Comment:
looks like the precommit didn't like this. can you run `uv run pre-commit run --all-files`?